c_src/term_to_value.cpp

```diff
@@ -16,6 +16,29 @@ namespace {
   }
 }
 
+bool nif::term_to_null(ErlNifEnv* env, ERL_NIF_TERM term, const duckdb::LogicalType& value_type, duckdb::Value& sink) {
+  if(!enif_is_atom(env, term))
```

Comment:
I think this is unnecessary check,  because the `enif_get_atom_length` below already checks if this atom or not